api/trusted-issuers-list.yaml

```diff
@@ -108,5 +139,27 @@ components:
             - 12
             - "GOLD_CUSTOMER"
             - "STANDARD_CUSTOMER"
```

Comment:
Why did you remove the reference for an embedded object here? The code still allows it or?